internal/proto/controller/storage/oplog/store/v1/oplog.proto

```diff
@@ -34,6 +34,14 @@ message Entry {
   // we are NOT storing this plain-text entry data in the db
   // @inject_tag: gorm:"-" wrapping:"pt,entry_data"
   bytes data = 8;
+
+  // the private id of the data key version used to encrypt the data.
+  // @inject_tag: gorm:"not_null"
+  string key_id = 9;
+
+  // the scope id associated with the key used to encrypt the data.
+  // @inject_tag: gorm:"not_null"
+  string scope_id = 10;
```

Comment:
Is this needed since it is filled in by a trigger?

Reply:
It's for reading the scope ID out of the entry, not for setting it. As you notice, setting the scope ID on insert will always be overwritten by the trigger.

Reply:
I'm happy to remove this if we don't think we'll need it

Reply:
I think it is fine to leave it